demo/hello.ts

```diff
@@ -461,6 +451,22 @@ export class HelloApp {
         { name: 'formly-wrapper-horizontal', component: FormlyWrapperHorizontalLabel, types: ['horizontalInput'] },
         { name: 'panel', component: FormlyPanelWrapper },
       ],
+      defaultOptions: [
+        {
```

Comment:
@mohammedzamakhan after looking at PR #232 seems like the direction in this PR is different then intended. Let me know if you want to me to close this PR and you will continue with #232 .